modular_skyrat/modules/aesthetics/lights/code/lighting.dm

```diff
@@ -96,3 +96,72 @@
 /obj/item/light/tube
 	icon = 'modular_skyrat/modules/aesthetics/lights/icons/lighting.dmi'
 
+
+// attack with item - insert light (if right type), otherwise try to break the light
```

Comment:
oh god, this isn't what I meant!

You just need to do . = ..() and then add your new functionality (the skyrat part). I am sure there are examples in the codebase about modularly adding more functionality to a proc.